frontend/src/tests/template-tests.js

```diff
@@ -20,7 +20,7 @@ import { base64ToBytes } from "../api/mockAPI/utils";
  * @param {Function} api.apiGET A function that when passed a route will return the get response
  * @param {Function} api.apiPOST A function that when passed a route and data, will return the post response
  */
-export function templatesTests(api) {
+export function templatesTests(api, callback) {
```

Comment:
Why do you need a callback? Test for and clean up the test file before you rely on its non-existence, instead.

Reply:
I thought it would be cleaner than just leaving the file after the test was over. I can keep the test file though.